toolkit/analysis_and_visualization/classification/assets/scripts/app.js

```diff
@@ -0,0 +1,637 @@
+        row.append($('<td>').attr('class', 'blue').text(label.match3));
+        row.append($('<td>').attr('class', 'blue').text(label.match4));
+        row.append($('<td>').attr('class', 'blue').text(label.match4));
+        row.append($('<td>').attr({
+            'class': 'blue filter-image ' + misclassifiedClass,
+            'data-id': label.id,
+            'data-type': 'misclassified'
+        }).text(label.misclassifiedTop1));
+
+        row.append($('<td>').html('<input id="id_"' + id + ' name="id[' + id + ']" type="checkbox" value="' + id + '" onClick="highlightRow(this);"></input>'));
+        myTableBody.append(row);
+    });
+
+
+}
+
+function filterLabelTable(rowNum, Datavar) {
+    var labelSummaryFiltered = [];
+    var allEmpty = true;
+    $('input[id ^= "fl_"]').each(function (element) {
+        var property = this.id.split("_")[1];
+        var value = this.value;
+        var compare = this.getAttribute('data-compare');
+        if (value) {
+            allEmpty = false;
+            labelSummary.forEach(function (label) {
+                var compareResult = false;
+
+                if (compare == 'contains') {
+                    compareResult = label[property].toLowerCase().includes(value.toLowerCase());
+
+                } else {
+                    compareResult = label[property] == value;
+                }
+                if (compareResult) {
+                    labelSummaryFiltered.push(label);
+                }
+            });
+        }
+    });
+    if (allEmpty) {
+        loadLabelSummary(labelSummary);
+    } else {
+        loadLabelSummary(labelSummaryFiltered);
+    }
+}
+
+function clearLabelFilter() {
+    $('input[id ^= "fl_"]').val('');
+
+    loadLabelSummary(labelSummary);
+}
+
+function highlightRow(element) {
+    var parentRow = element.parentElement.parentElement;
+    parentRow.classList.toggle("highlight-row");
+    console.log(element.parentElement.parentElement);
+
+}
+
+
+//Create copy of imageSummary
+var imageSummaryFiltered = JSON.parse(JSON.stringify(imageSummary));
+
+function clearResultFilter() {
+    $('input[id ^= "fli_"]').val('');
+    imageSummaryFiltered = JSON.parse(JSON.stringify(imageSummary));
+    loadImageResults(imageSummaryFiltered);
+    updateItems();
+}
+
+function filterResultTable(e) {
+    imageSummaryFiltered = [];
+    var allEmpty = true;
+    if (e) {
+        var code = (e.keyCode ? e.keyCode : e.which);
+
+    }
+    var combinationOp = $("input:radio[name='filterType']:checked").val();
+    var notOp = $('#not-op').is(":checked");
+
+
+    var filters = {};
+    var count = 0;
+
+    // if (combinationOp == 'and')
+    //     filterSource = imageSummaryFiltered
+    // else
+    //     filterSource = imageSummary
+
+    $('input[id ^= "fli_"]').each(function (element) {
+        var allNames = this.id.split("_");
+        var property = allNames[1];
+        var property_index = null;
```

Comment:
Codacy found an issue: [Identifier 'property_index' is not in camel case.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)